### PR TITLE
applications: matter_bridge: Disable WPA3 for memory profiling config

### PR DESCRIPTION
**Pull request by adigie**
Disable WPA3 to reduce FLASH usage.

### CODE REVIEW

**Review by ArekBalysNordic — APPROVED** [review]

**Review by doublemis1 — APPROVED** [review]

File: applications/matter_bridge/sample.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ tests:
     extra_args: CONFIG_BRIDGE_GENERIC_SWITCH_BRIDGED_DEVICE=n
       CONFIG_BRIDGE_ONOFF_LIGHT_SWITCH_BRIDGED_DEVICE=y
       CONFIG_CHIP_MEMORY_PROFILING=y CONFIG_BRIDGED_DEVICE_BT=y
-      CONFIG_BRIDGE_MIGRATE_VERSION_1=n
+      CONFIG_BRIDGE_MIGRATE_VERSION_1=n CONFIG_WIFI_NM_WPA_SUPPLICANT_WPA3=n
```
Comment (doublemis1) [inline applications/matter_bridge/sample.yaml:102]: Do you know what is the memory reduction due to disabling WPA3?

**Comment by doublemis1** [comment]
backport to 2.9 branch label needed?

**Review by LuDuda — APPROVED** [review]

**Comment by doublemis1** [comment]
Please remove the records regarding bridge and light switch from quarantine.
scripts/quarantine.yaml

File: applications/matter_bridge/sample.yaml (hunk 1 above)
Comment (adigie) [reply applications/matter_bridge/sample.yaml:102]: 15.6KB of FLASH, 320 bytes of RAM.

**Review by ArekBalysNordic — APPROVED** [review]

**Comment by adigie** [comment]
@nrfconnect/ncs-test-leads Please review.

**Review by mariusz-nordicsemi — APPROVED** [review]

**Pull request merged**